examples/TestReceiveAV/README.md

```diff
@@ -0,0 +1,45 @@
+#Usage
+
+ - Generate an X509 certificate and key for the web socket server using the instructions below,
+ - Build and run the solution file,
+ - Open the `webrtc.html` page in a WebRTC enabled browser.
+ - Click the `Start` button in the Web Browser.
+ - If successful a new Window should pop-up displaying the Video stream received from the browser and the audio should be played on the default system speaker.
+
+ #Certificate Instructions
+
+ A web socket connection is used to exchange the SDP offer and answer as well as ICE candidates. The web socket connection will only work if listening port on the test application can be accessed by the browser. It should work correctly if both are on the same development machine or local network.
+
+ A PKCS12 archive file which contains an X509 certificate with a common name of "localhost" and a private key is required. The archive file is needed for the web socket server and is suitable for use when the web socket connection is between a browser and the test application running on the same machine. The instructions use to generate the certificate using openssl are below:
+
+ ````
+openssl req -config req.conf -x509 -newkey rsa:4096 -keyout private/localhost.pem -out localhost.pem -nodes -days 3650

```

Comment:
I tried running this locally passing the req.conf example below, I got
```
Error Loading extension section x509_ext
19992:error:2207507C:X509 V3 routines:v2i_GENERAL_NAME_ex:missing value:crypto/x509v3/v3_alt.c:512:
19992:error:22098080:X509 V3 routines:X509V3_EXT_nconf:error in extension:crypto/x509v3/v3_conf.c:47:name=subjectAltName, value=localhost, IP:127.0.0.1, IP:::1
```

Maybe there is something missing/wrong in the sample?

Reply:
It worked after rewriting the subjectAltName line as in [this sample](https://gist.github.com/andrewconnell/6c2c14e80ef45b232d11e2f4706489b5) - I am not familiar with the syntax so I am not sure why, but it might give you an idea.

Btw the readme should mention that you need to assign an empty password to the PKCS12 file, or the server program won't load it as it is right now.

Reply:
I've updated the openssl config and instructions for generating the localhost self signed certificate.